tests/test_dmlrunner.py

```diff
@@ -69,3 +69,22 @@ def test_dmlrunner_validate_job_returns_stats(config_manager):
     val_stats = results['val_stats']
     assert result.job.job_type is JobTypes.JOB_VAL.name
     assert type(val_stats) == dict
+
+def test_dmlrunner_initialize_job_weights_can_be_serialized(config_manager):
+    model_json = make_model_json()
+    runner = DMLRunner(config_manager)
+    initialize_job = make_initialize_job(model_json)
+    initial_weights = runner.run_job(initialize_job).results['weights']
+    same_weights = deserialize_weights(serialize_weights(initial_weights))
+    assert(np.shape(same_weights) == np.shape(initial_weights))
+    assert all([np.count_nonzero(arr)==0 for arr in np.subtract(same_weights, initial_weights)])
```

Comment:
We might have talked about this offline, but why not directly check if both arrays are equal? Try [this](https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.array_equal.html), and I believe this reduces the complexity of these two assert statements.

Reply:
Sorry, try [this](https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.allclose.html) . 

```
assert all(np.allclose(arr1, arr2) for arr1,arr2 in zip(same_weights, initial_weights)) 
```

Reply:
Makes it more clear what the assert statements are trying to accomplish.

Reply:
Done